public/countries.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
```

Comment:
>I think it would be better to have all the countries in Divs so that they can be inline and the explore button closer to the content on large screens.

Reply:
And keep the introduction "These countries have their tax and benefit system model." etc. on two columns?